Mailbox/Sizes.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+
+namespace Mailbox
+{
+    [Flags]
+    public enum Sizes
+    {
+        Undefined = 0b0000,
+        Small     = 0b0001,
+        Medium    = 0b0010,
+        Large     = 0b0100,
+        Premium   = 0b1000,
+    }
+
+    public static class SizesExtensions
+    {
+        public static Sizes Size(this Sizes sizes) => sizes & ~Sizes.Premium;
+
+        public static bool Premium(this Sizes sizes) => sizes.HasFlag(Sizes.Premium);
```

Comment:
Similar to the coding standard for properties, you typically would do the same for methods.
`DO favor prefixing Boolean properties with “Is,” “Can,” or “Has,” when that practice adds value.`

```suggestion
        public static bool IsPremium(this Sizes sizes) => sizes.HasFlag(Sizes.Premium);
```